project_charters/opentimelineio-charter.md

```diff
@@ -0,0 +1,67 @@
+
+## 5. Community Assets
+  * **a.** LF Projects shall hold title to all trade or service marks used by the Project ("Project Trademarks"), whether based on common law or registered rights. Project Trademarks shall be transferred and assigned to LF Projects to hold on behalf of the Project. Any use of any Project Trademarks by participants in the Project shall be in accordance with the license from LF Projects and inure to the benefit of LF Projects.
+  * **b.** The Project shall, as permitted and in accordance with such license from LF Projects, develop and own all Project GitHub and social media accounts, and domain name registrations created by the Project community.
+  * **c.** Under no circumstances shall LF Projects be expected or required to undertake any action on behalf of the Project that is inconsistent with the tax-exempt status or purpose, as applicable, of LFP, Inc. or LF Projects, LLC.
+
+## 6. General Rules and Operations.
+  * **a.** The Project will:
+    * **i.** engage in the work of the project in a professional manner consistent with maintaining a cohesive community, while also maintaining the goodwill and esteem of LF Projects, LFP, Inc. and other partner organizations in the open source software community; and
+    * **ii.** respect the rights of all trademark owners, including any branding and trademark usage guidelines.
+
+## 7. Intellectual Property Policy
+  * **a.** Participants acknowledge that the copyright in all new contributions shall be retained by the copyright holder as independent works of authorship and that no contributor or copyright holder will be required to assign copyrights to the Project.
+  * **b.** Except as described in Section 7.c., all code contributions to the Project are subject to the following:
+    * **i.** All new inbound code contributions to the Project must be made using the **[Apache License, Version 2.0 (available here: https://www.apache.org/licenses/LICENSE-2.0)]** (the "Project License").
+    * **ii.** All new inbound code contributions must:
+      1. be made pursuant to a duly executed Project Contribution License Agreement (the "CLA"), available on the Project's web site; and
+      2. be accompanied by a Developer Certificate of Origin (http://developercertificate.org) sign-off in the source code system that is submitted through a TSC-approved contribution process which will bind the authorized contributor and, if not self-employed, their employer to the applicable license;
+    * **iii.** All outbound code will be made available under the Project License, with the exception of code repositories designated by the TSC to be used as a template repository, which will be made available under a choice of either the Project License or the MIT License ( available here: https://spdx.org/licenses/MIT.html )
```

Comment:
What does this mean?

Reply:
The project is creating a template for those in the community that wish to develop plugins, and doesn't want to enforce a specific license on those plugins. By providing an option on the license, with one option being MIT, that gives the downstream community as much latitude as possible to relicense as appropriate.

Let me know if that answers your question.

Reply:
Yes, thanks, I just didn't understand what "template repository" was referring to.